### PR TITLE
More Refactoring of Reportpages + Localization fix

### PR DESCRIPTION
**Pull request by Skywalker-11**
Adds an error message if cache directory is not writable
Fix de localization
Fix localization for top10recipdomqt40
More refactoring to move functionality from report pages to functions.php
Removes Mailscanner logo from report pages

### CODE REVIEW

**Comment by stefaweb** [comment]
> 'errorcachedirnotwritable03' => 'Error: cache directory %s not writable',

'errorcachedirnotwritable03' => 'Erreur : impossible d\'écrire dans le répertoire de cache %s',

**Comment by stefaweb** [comment]
> remove mailscanner log from page

Can may be replaced by an "optional" user logo?

**Comment by Skywalker-11** [comment]
> Can may be replaced by an "optional" user logo?
Why should we set a logo on the report sites only?

**Comment by stefaweb** [comment]
Forgot my question. I was thinking that you was talking of "quarantine report"...

**Comment by stefaweb** [comment]
Could you fix this. We have plenty of space to ovoid the use of "min.".

en.php and others:
```
    '1minute03' => '1 minute:',
    '5minutes03' => '5 minutes:',
    '15minutes03' => '15 minutes:',
```
fr.php
```
    '1minute03' => '1 minute :',
    '5minutes03' => '5 minutes :',
    '15minutes03' => '15 minutes :',
```

de.php
```
    '1minute03' => '1 Minute.:',
    '5minutes03' => '5 Minutes:',
    '15minutes03' => '15 Minutes:',
```

**Comment by Skywalker-11** [comment]
that is unrelated to this pr and should be a seperate one

File: mailscanner/languages/it.php [hunk 1]
```diff
@@ -670,7 +671,7 @@
 
     // 40-rep_top_recipient_domains_by_quantity.php
     'toprecipdomqt40' => 'Top Recipients Domains by Quantity',
-    'top10recipdomqt40' => 'Top 10 Recipients Domains by Volume',
+    'top10recipdomqt40' => 'Top 10 Recipients Domains by Quantinty',
```
Comment (endelwar) [inline mailscanner/languages/it.php]: typo

File: mailscanner/languages/nl.php [hunk 2]
```diff
@@ -670,7 +671,7 @@
 
     // 40-rep_top_recipient_domains_by_quantity.php
     'toprecipdomqt40' => 'Top Recipients Domains by Quantity',
-    'top10recipdomqt40' => 'Top 10 Recipients Domains by Volume',
+    'top10recipdomqt40' => 'Top 10 Recipients Domains by Quantinty',
```
Comment (endelwar) [inline mailscanner/languages/nl.php]: typo

File: mailscanner/languages/pt_br.php [hunk 3]
```diff
@@ -670,7 +671,7 @@
 
     // 40-rep_top_recipient_domains_by_quantity.php
     'toprecipdomqt40' => 'Top Recipients Domains by Quantity',
-    'top10recipdomqt40' => 'Top 10 Recipients Domains by Volume',
+    'top10recipdomqt40' => 'Top 10 Recipients Domains by Quantinty',
```
Comment (endelwar) [inline mailscanner/languages/pt_br.php]: typo

**Comment by endelwar** [comment]
Is Graph.js in this PR a leftover? can't see any linking in code

**Comment by Skywalker-11** [comment]
yes will fix

**Comment by endelwar** [comment]
Manually rebased and merged in 5468ca9ac0146107f2f189848ad63e19f3f59d69

**Pull request closed without merging**